orca/configs/single_server.yml

```diff
@@ -13,5 +13,7 @@ hosts:
             - monitoring-proxy
             - monitoring-proxy-db
             - search
+            - teleport-agent-test
```

Comment:
Lets not add them to the default configuration as it will slow down the development. We should only add things which are working without any problems and fully integrated (learning from my mistakes with the vault)